### ISSUE TITLE
autoscaling.Tags PropagateAtLaunch value is output as string, not boolean

### ISSUE THREAD

**Issue by mikeshimkus** [cloudtools/troposphere#1871]
Troposphere v2.6.2

When specifying the following for autoscaling.Tags and TROPO_REAL_BOOL=true, the tags' PropagateAtLaunch value is output as a string, not a boolean (which is what cfn-lint expects).  All other python object booleans are properly output as JSON booleans, just not this one.

```
Tags=autoscaling.Tags(
                Name=Join("", ["Autoscale Instance: ", Ref("deploymentName")]),
                Application=Ref("application"),
                Environment=Ref("environment"),
                Group=Ref("group"),
                Owner=Ref("owner"),
                Costcenter=Ref("costcenter")
            )
```

Actual output:
```json
  "Tags": [
     {
      "Key": "Application",
      "PropagateAtLaunch": "true",
      "Value": {
       "Ref": "application"
      }
     },
     {
      "Key": "Costcenter",
      "PropagateAtLaunch": "true",
      "Value": {
       "Ref": "costcenter"
      }
     },
     {
      "Key": "Environment",
      "PropagateAtLaunch": "true",
      "Value": {
       "Ref": "environment"
      }
     },
     {
      "Key": "Group",
      "PropagateAtLaunch": "true",
      "Value": {
       "Ref": "group"
      }
     },
     {
      "Key": "Name",
      "PropagateAtLaunch": "true",
      "Value": {
       "Fn::Join": [
        "",
        [
         "Autoscale Instance: ",
         {
          "Ref": "deploymentName"
         }
        ]
       ]
      }
     },
     {
      "Key": "Owner",
      "PropagateAtLaunch": "true",
      "Value": {
       "Ref": "owner"
      }
     }
    ]
```

Expected output:
```json
  "Tags": [
     {
      "Key": "Application",
      "PropagateAtLaunch": true,
      "Value": {
       "Ref": "application"
      }
     },
     {
      "Key": "Costcenter",
      "PropagateAtLaunch": true,
      "Value": {
       "Ref": "costcenter"
      }
     },
     {
      "Key": "Environment",
      "PropagateAtLaunch": true,
      "Value": {
       "Ref": "environment"
      }
     },
     {
      "Key": "Group",
      "PropagateAtLaunch": true,
      "Value": {
       "Ref": "group"
      }
     },
     {
      "Key": "Name",
      "PropagateAtLaunch": true,
      "Value": {
       "Fn::Join": [
        "",
        [
         "Autoscale Instance: ",
         {
          "Ref": "deploymentName"
         }
        ]
       ]
      }
     },
     {
      "Key": "Owner",
      "PropagateAtLaunch": true,
      "Value": {
       "Ref": "owner"
      }
     }
    ]
```

**Comment by marinpurgar** [comment]
Looks like the autoscaling.tags() was forgotten when converting all boolean parameters from str to "real boolean" with 
TROPO_REAL_BOOL=true. https://github.com/cloudtools/troposphere/pull/1409
The https://github.com/cloudtools/troposphere/pull/1874 fixes this behavior.

**Comment by markpeek** [comment]
Should be fixed via the merge of #1874. Thanks @marinpurgar!
